version.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.0.4</VersionPrefix>
+    <VersionPrefix>2.0.5</VersionPrefix>
```

Comment:
Oh wait, wasn't the release we just did 2.0.5? And Feb is 2.0.6?

Reply:
Yup. I'm waiting to merge this until we get those changes into the public branch. Then I'll bump to 2.0.6

Reply:
Ah ok thanks for clarifying.